### ISSUE TITLE
Paragraph on "We’re transparent about our operational processes"

### ISSUE THREAD

**Issue by itrich** [SovereignCloudStack/open-operations-manifesto#6]

**Comment by fkr** [comment]
@heinritzkat and @cedegi came up with an excellent proposal:

```
We share our internal processes for the sake of transparency. We firmly believe that transparency leads to better and more reliable processes. We are convinced that publishing well-documented operational processes offers a great opportunity to gather feedback and inspiration from the wide public. 
 
Likewise, we are convinced that us sharing our operational experiences can be of inspiration and benefit for others. In today's business environment, most companies are facing the same challenges. By sharing knowledge the same way the open source community shares software, a common knowledge base can become a reality, allowing everybody to benefit. 
 
We aim to get away from being a big black box and become a source of shared knowledge. We consider the disclosure of internal processes as an essential step towards this direction.
```

I'll see with @heinritzkat to make a PR.

**Comment by itrich** [comment]
Fixed by #9
